src/test/java/com/wikia/webdriver/testcases/discussions/CreatingPostTests.java

```diff
@@ -240,6 +250,23 @@ private CategoryPill fillPostCategoryWith(final PostsCreator postsCreator, final
     return categoryPill;
   }
 
+  private String addLinkToDescription(final PostsCreator postsCreator, final String description) throws MalformedURLException {
```

Comment:
I think it is proper place. `BasePostsCreator` has `url`, in method. This class user responsibility to proper create url.